### DISCUSSION TITLE
Vercel deployment failed - runs well on local.

### DISCUSSION THREAD

**Discussion by tomkochi** [vercel/community#176] [Help] [answered]
Hey, It's about nextjs/prismic.
Yarn dev, yarn built, yarn start - all works fine locally. No errors, nothing. But when it's in Vercel, it all ruined.

![Screenshot 2021-11-30 at 9 09 16 am](https://user-images.githubusercontent.com/33484729/143983180-871a8401-ea35-4148-a8a1-99bdc08217e7.png)

Please help me out. I am past deadloine already.

And, it works flawlessly on Heroku.

**Comment by Pieparker** [comment, marked as answer]
Hey there! This conversation appears to be quiet but I will share [these words](https://www.reddit.com/r/nextjs/comments/j4i5c1/comment/g7jeoo5) from our very own @leerob verbatim, for posterity:

>You can view your error logs by going to your Vercel dashboard, clicking on your project, and then clicking "View Function Logs". This will show errors happening in realtime. They are not persisted (they're streaming logs). Refresh the page, and you'll see your 500 error. That should tell you what the issue is.

To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
